### PR TITLE
added release notes

### PR DESCRIPTION
**Pull request by kellyselden**
* beta info
* windows info

comments appreciated @stefanpenner @rwjblue

### CODE REVIEW

**Comment by kellyselden** [comment]
friendly ping @stefanpenner @rwjblue

**Comment by stefanpenner** [comment]
thanks for the ping, will review

**Comment by rwjblue** [comment]
Looks good to me

File: RELEASE.md [hunk 1]
```diff
@@ -41,22 +51,29 @@ and failures we do not want our users to endure any extra pain.
...
+  * if beta release
+    * `npm install -g ember-cli@beta`
 * ensure version is as expected `ember version`
 * ensure new project generates
 * ensure old project upgrades nicely
 
 Tag the release
-1. Under `Releases` on GitHub choose `Draft New Release`
+
+* Under `Releases` on GitHub choose `Draft New Release`
```
Comment (kellyselden) [inline RELEASE.md]: I'm going to undo this line, didn't mean to lose the numbering

Comment (kellyselden) [inline RELEASE.md]: done

File: RELEASE.md [hunk 2]
```diff
@@ -64,3 +81,14 @@ Update the site
 1. check out ember-cli/ember-cli.github.io
 * update `_config.yml` version
 * update `_posts/2012-01-01-changelog.md`
+
+Announce release!
+
+1. on Twitter
```
Comment (stefanpenner) [inline RELEASE.md]: we should make a release tweet bot someday

**Comment by stefanpenner** [comment]
sorry for letting this sit, looks great!

**Pull request merged**